pmd-apex/src/test/java/net/sourceforge/pmd/lang/apex/metrics/ApexMetricsVisitorTest.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.apex.metrics;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import net.sourceforge.pmd.lang.LanguageRegistry;
+import net.sourceforge.pmd.lang.LanguageVersionHandler;
+import net.sourceforge.pmd.lang.apex.ApexLanguageModule;
+import net.sourceforge.pmd.lang.apex.ast.ASTMethod;
+import net.sourceforge.pmd.lang.apex.ast.ApexNode;
+import net.sourceforge.pmd.lang.apex.ast.ApexParserTest;
+import net.sourceforge.pmd.lang.apex.ast.ApexParserTestHelpers;
+import net.sourceforge.pmd.lang.apex.ast.ApexParserVisitorAdapter;
+import net.sourceforge.pmd.lang.apex.metrics.signature.ApexOperationSigMask;
+
+import apex.jorje.semantic.ast.compilation.Compilation;
+
+/**
+ * @author Clément Fournier
+ */
+public class ApexMetricsVisitorTest extends ApexParserTest {
+
+    @Test
+    public void testProjectMirrorNotNull() {
+        assertNotNull(ApexMetrics.getFacade().getProjectMirror());
+    }
+
+
+    @Test
+    public void testOperationsAreThere() {
+        ApexNode<Compilation> acu = parseAndVisitForString("public with sharing class MetadataDeployController \n"
```

Comment:
You might externalize the source code into a test resource. This makes it more readable.
You can easily load the file then with `IOUtils.toString(ApexMetricsVisitorTest.class.getResourceAsStream("MetadataDeployController.cls")`.